<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>778</width>
    <height>689</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BIPy</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
	background: #262d37;
	color: white;
}

QPushButton{
	border-radius: 20%;;
	background: rgba(0, 71, 133,0.4);
	color: white;
	border: 2px solid #000;
}

QPushButton:hover{
	background: rgba(0, 71, 133,0.3);
}

QPushButton:pressed{
	background:rgba(0, 71, 133,0.2);
}

QLCDNumber{
  background-color: rgb(0, 0, 0);
  border: 2px solid rgb(113, 113, 113);
  border-width: 2%;
  border-radius: 20%;
  color: rgb(255, 255, 255);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>38</red>
         <green>45</green>
         <blue>55</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>38</red>
         <green>45</green>
         <blue>55</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>38</red>
         <green>45</green>
         <blue>55</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>38</red>
         <green>45</green>
         <blue>55</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>38</red>
         <green>45</green>
         <blue>55</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>38</red>
         <green>45</green>
         <blue>55</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>38</red>
         <green>45</green>
         <blue>55</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>38</red>
         <green>45</green>
         <blue>55</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>38</red>
         <green>45</green>
         <blue>55</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>20</number>
       </property>
       <property name="topMargin">
        <number>20</number>
       </property>
       <property name="rightMargin">
        <number>20</number>
       </property>
       <property name="bottomMargin">
        <number>20</number>
       </property>
       <property name="horizontalSpacing">
        <number>20</number>
       </property>
       <property name="verticalSpacing">
        <number>5</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Instrução</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLCDNumber" name="program_counter">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="digitCount">
          <number>4</number>
         </property>
         <property name="mode">
          <enum>QLCDNumber::Hex</enum>
         </property>
         <property name="value" stdset="0">
          <double>0.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="text">
          <string>PC</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLCDNumber" name="instruct_counter">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="digitCount">
          <number>1</number>
         </property>
         <property name="mode">
          <enum>QLCDNumber::Hex</enum>
         </property>
         <property name="value" stdset="0">
          <double>0.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLCDNumber" name="acumulador">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="digitCount">
          <number>4</number>
         </property>
         <property name="mode">
          <enum>QLCDNumber::Hex</enum>
         </property>
         <property name="value" stdset="0">
          <double>0.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QFrame" name="frame_4">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="font">
             <font>
              <pointsize>20</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Acumulador</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_5">
            <property name="styleSheet">
             <string notr="true">QLabel{
	border: 1px solid black;
	border-radius: 10%;
	background: rgba(0, 71, 133,0.4);
}</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="label_4">
               <property name="styleSheet">
                <string notr="true">QLabel{
	background:rgba(0, 71, 133, 0.4);
}</string>
               </property>
               <property name="text">
                <string>Hexadecimal</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_5">
               <property name="styleSheet">
                <string notr="true">QLabel{
	background:rgba(0, 71, 133, 0.1);
}</string>
               </property>
               <property name="text">
                <string>Decimal</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="buttonBox"/>
    </item>
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>20</number>
       </property>
       <property name="leftMargin">
        <number>20</number>
       </property>
       <property name="topMargin">
        <number>20</number>
       </property>
       <property name="rightMargin">
        <number>20</number>
       </property>
       <property name="bottomMargin">
        <number>20</number>
       </property>
       <item>
        <widget class="QPushButton" name="reset_button">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>60</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="toolTip">
          <string>Reseta o runtime do processador</string>
         </property>
         <property name="statusTip">
          <string>Reseta o runtime do processador</string>
         </property>
         <property name="whatsThis">
          <string/>
         </property>
         <property name="text">
          <string>Reset</string>
         </property>
         <property name="shortcut">
          <string>Ctrl+R</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="halt_check">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>60</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="toolTip">
          <string>Step automático</string>
         </property>
         <property name="toolTipDuration">
          <number>-1</number>
         </property>
         <property name="statusTip">
          <string>Step automático</string>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">
QPushButton:checked{ 
	background-color: rgba(0, 71, 133,0.8);
 }</string>
         </property>
         <property name="text">
          <string>Halt</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="step_button">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>60</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="toolTip">
          <string>Executar a próxima instrução</string>
         </property>
         <property name="statusTip">
          <string>Executar a próxima instrução</string>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Step</string>
         </property>
         <property name="shortcut">
          <string>Space</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_3">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>50</number>
       </property>
       <property name="leftMargin">
        <number>20</number>
       </property>
       <property name="topMargin">
        <number>20</number>
       </property>
       <property name="rightMargin">
        <number>20</number>
       </property>
       <property name="bottomMargin">
        <number>20</number>
       </property>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>75</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="toolTip">
          <string>Abrir memoria de programa</string>
         </property>
         <property name="toolTipDuration">
          <number>0</number>
         </property>
         <property name="statusTip">
          <string>Abrir memoria de programa</string>
         </property>
         <property name="text">
          <string>Memoria de Programa</string>
         </property>
         <property name="shortcut">
          <string>Ctrl+P</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>75</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="toolTip">
          <string>Abrir memoria de dados</string>
         </property>
         <property name="statusTip">
          <string>Abrir memoria de dados</string>
         </property>
         <property name="text">
          <string>Memoria de Dados</string>
         </property>
         <property name="shortcut">
          <string>Ctrl+D</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>778</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArquivo">
    <property name="title">
     <string>Arquivo</string>
    </property>
    <widget class="QMenu" name="menuAbrir">
     <property name="title">
      <string>Carregar</string>
     </property>
     <addaction name="actionMemoria_de_Dados"/>
     <addaction name="actionMemoria_de_Programa"/>
    </widget>
    <widget class="QMenu" name="menuSalvar">
     <property name="title">
      <string>Salvar</string>
     </property>
     <addaction name="actionMemoria_de_Dados_2"/>
     <addaction name="actionMemoria_de_Programa_2"/>
    </widget>
    <addaction name="menuSalvar"/>
    <addaction name="menuAbrir"/>
   </widget>
   <widget class="QMenu" name="menuConfigura_o">
    <property name="title">
     <string>Configuração</string>
    </property>
    <widget class="QMenu" name="menuAlterar_modo_do_acumulador">
     <property name="title">
      <string>Alterar modo do acumulador</string>
     </property>
     <addaction name="actionHexadecimal"/>
     <addaction name="actionDecimal"/>
    </widget>
    <addaction name="actionSetar_Clock"/>
    <addaction name="menuAlterar_modo_do_acumulador"/>
   </widget>
   <addaction name="menuArquivo"/>
   <addaction name="menuConfigura_o"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbrir_Memoria_de_Dados">
   <property name="text">
    <string>Abrir Memoria de Dados</string>
   </property>
  </action>
  <action name="actionMemoria_de_Dados">
   <property name="text">
    <string>Memoria de Dados</string>
   </property>
  </action>
  <action name="actionMemoria_de_Programa">
   <property name="text">
    <string>Memoria de Programa</string>
   </property>
  </action>
  <action name="actionMemoria_de_Dados_2">
   <property name="text">
    <string>Memoria de Dados</string>
   </property>
  </action>
  <action name="actionMemoria_de_Programa_2">
   <property name="text">
    <string>Memoria de Programa</string>
   </property>
  </action>
  <action name="actionSetar_Clock">
   <property name="text">
    <string>Setar Clock</string>
   </property>
   <property name="statusTip">
    <string>Altera a velocidade do clock</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="actionHexadecimal">
   <property name="text">
    <string>Hexadecimal</string>
   </property>
   <property name="statusTip">
    <string>Alternar o modo de display do acumulador para Hexadecimal</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+H</string>
   </property>
  </action>
  <action name="actionDecimal">
   <property name="text">
    <string>Decimal</string>
   </property>
   <property name="statusTip">
    <string>Alternar o modo de display do acumulador para Decimal</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+D</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
